src/others/sysmon/lv_sysmon.c

```diff
@@ -163,7 +167,7 @@ static void perf_update_timer_cb(lv_timer_t * t)
     if(!sysmon_perf.inited && lv_display_get_default()) {
         lv_display_add_event_cb(lv_display_get_default(), perf_monitor_disp_event_cb, LV_EVENT_ALL, NULL);
 
-        lv_obj_t * obj1 = lv_sysmon_create(lv_layer_sys());
```

Comment:
I vote for renaming to `lv_sysmon_show` to not follow the widget-like API for non-widgets.

What do you think?

Reply:
works for me. The sysmon is not a widget so it really shouldn't use the same kind of naming convention as a widget.

Reply:
OK, I will add show and hide functions (for both performance and memory) in the next days.

Reply:
Functions are added